base/shell/explorer/taskswnd.cpp

```diff
@@ -1382,24 +1381,31 @@ class CTaskSwitchWnd :
         m_TaskBar.EndUpdate();
     }
 
-    BOOL CALLBACK EnumWindowsProc(IN HWND hWnd)
+    BOOL IsTaskWnd(HWND hWnd) const
     {
-        /* Only show windows that still exist and are visible and none of explorer's
-        special windows (such as the desktop or the tray window) */
         if (::IsWindow(hWnd) && ::IsWindowVisible(hWnd) &&
             !m_Tray->IsSpecialHWND(hWnd))
         {
-            DWORD exStyle = ::GetWindowLong(hWnd, GWL_EXSTYLE);
+            DWORD exStyle = ::GetWindowLongPtr(hWnd, GWL_EXSTYLE);
```

Comment:
Either convert `exStyle` to a `LONG_PTR`, or cast the returned value to `DWORD`.

Reply:
Casted.